lib/utils/resolveObject.js

```diff
@@ -31,7 +30,7 @@ let updated_object, regex, options;
 export default function resolveObject(object, opts) {
   options = Object.assign({}, defaults, opts);
 
-  updated_object = cloneDeep(object); // This object will be edited
+  updated_object = structuredClone(object); // This object will be edited
```

Comment:
Just a heads up `structuredClone` may only be supported for Node 17 & up.
https://developer.mozilla.org/en-US/docs/Web/API/structuredClone

I think this project still expects to be run on Node 12. Maybe the `engines.node` in package.json will need to be updated to 18 or 20.